lib/svgo/config.js

```diff
@@ -73,8 +118,11 @@ function preparePluginsArray(plugins) {
                 plugin = setupCustomPlugin(key, item[key]);
 
             } else {
+              if (!pluginMap[key]) {
+                  throw new Error("Unknown plugin '" + key + "'");
```

Comment:
This is to maintain a similar behavior to what happens when you try to `require` a plugin that doesn't exist. Now that the plugins are not being `require`d in this function, this Error maintains backwards compatibility of what should happen for non-existent plugins.